Views: Add Notifier view

xenophenes opened the pull request:
- Add 'FormHeading' component
- Add placeholder content corresponding to wireframe
- TODO: Integrate search & notify forms with API
- TODO: Add appropriate CSS/styles

jmezzacappa reviewed (changes requested):
Looks good!
My only request is to update the JSDoc comments that were copied over from the `Categories` view, to make it easier for other developers.

xenophenes commented:
@jmezzacappa Hm, I'm looking through the added pages under `/Notifier/` and am not seeing any inaccurate comments. Is it just the main `/Notifier/index.jsx` file that you were thinking could use the clarification? I do see this line:

```
// TODO: Send parcel updates to backend
``` 

which I did rephrase slightly to:

```
// TODO: Send notification about selected parcels to backend
```

Let me know if you had anything specific in mind & I'm happy to update further!

jmezzacappa commented on src/views/Notifier/components/NotifyForm/index.jsx:
```diff
@@ -0,0 +1,81 @@
+/** @typedef {import('api').Parcel} Parcel */
+
+/**
+ * Form for editing a parcel's risk
```
This form is not used for editing a parcel's risk. Its purpose is to search for surrounding parcels within a specified radius to be notified.

xenophenes committed:
Views: Add Notifier view
Add 'FormHeading' component
Add placeholder content corresponding to wireframe
TODO: Integrate search & notify forms with API
TODO: Add appropriate CSS/styles
